ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala

```diff
@@ -106,19 +111,18 @@ object LedgerApiTestTool {
         sys.exit(1)
       })
 
-    val runner = new LedgerTestSuiteRunner(
-      LedgerSessionConfiguration(
-        config.participants,
-        config.shuffleParticipants,
-        config.tlsConfig,
-        config.loadScaleFactor,
-        config.partyAllocation,
-      ),
-      testsToRun.values.toVector,
-      identifierSuffix,
-      config.timeoutScaleFactor,
-      config.concurrentTestRuns,
-    )
+    val runner =
+      if (performanceTestsToRun.nonEmpty)
+        newLedgerSuiteRunner(
+          config,
+          performanceTestsToRun.values,
+          1
```

Comment:
Please name this argument. If I came back to this in a week, I would have no idea what it's for.